frontend/src/pages/Login.tsx

```diff
@@ -0,0 +1,73 @@
+import { useState } from "react";
+import { Link } from "react-router-dom";
+
+export function Login() {
+    const [email, setEmail] = useState("");
+    const [password, setPassword] = useState("");
+
+    const handleLogin = (event: React.FormEvent<HTMLFormElement>) => {
+        event.preventDefault();
+
+        fetch('http://localhost:8080/login', {
+            method: 'POST',
+            headers: {
+                'Accept': 'application/json, text/plain, */*',
+                'Content-Type': 'application/json'
+            },
+            body: JSON.stringify({ email, password })
+        })
+        .then(async res => {
+            // Ellenőrizd a válasz státuszát
```

Comment:
![image](https://github.com/user-attachments/assets/53ef4402-1c1e-4330-8d5e-9c071d6b215e)
What is this gibberish languange?